pom.xml

```diff
@@ -126,6 +131,16 @@
                   <exclude>webapp</exclude>
                 </excludes>
               </generator>
+              <enricher>
+                <includes>
+                  <include>wildfly-swarm-health-check</include>
+                </includes>
+                <config>
+                  <wildfly-swarm-health-check>
+                    <path>/</path>
```

Comment:
Yes, that's how it's done in Vert.x. I can change it back to /api/greeting, but for health check it doesn't make any difference. As long as HTTP status is under 500 all is good.